packages/kaizen-tg/_templates/drupal-9-theme/new/.stylelintignore

```diff
@@ -2,3 +2,6 @@
 to: <%= h.src() %>/<%= h.changeCase.lower(name) %>/.stylelintignore
 ---
 /color/*.css
+node_modules/**/*.css
+dist/**/*.css
```

Comment:
why this line ignored?

Reply:
Case of this check, was met after including the file with stylelint diable inside, but after it was gathered in the file - it was still checking in dist/css.
Don't see a reason to check dist files - if we already checked in on storybook side, on Drupal CSS side.

Reply:
gonna continue fixing in kaizen-breakpoints.

Reply:
This case dist/**/*.css 
could be related to including some files with style lint ignore  and it is already gathered in the "dist" but there could be some issues with the check and order.
So exclude check in the "dist" folder - useful.